### PR TITLE
Restaurant webpage

### PR DESCRIPTION
**Pull request by ABDELLANI-Youcef**
## Restaurant webpage
In this project we:
- create index.html page.
- add picture and presentation of the restaurant.
- add tabs for Menus and Contacts.
- add styling to the webpage

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @ABDELLANI-Youcef  👋

**Good job here. 👌**

However, let's make a few fixes.

- [x] Please check the comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "This is a javascript project in main curriculum in Microverse.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (mikenath223) [inline package.json]: - [ ] Please update the script section here with the necessary scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  'facebook: youcefabdellani',
+  'official site: youcefrestaurant.dz',
+];
+
+const ulist = document.createElement('ul');
+for (let i = 0; i < contacts.length; i += 1) {
+  const item = document.createElement('li');
+  item.textContent = contacts[i];
+  ulist.appendChild(item);
+}
```
Comment (mikenath223) [inline src/contact.js]: - [ ] Please aim to use ES6 syntax more. Kindly replace this forLoop with a map function:
`contacts.map((item, index) => { code here })`

File: src/menu.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  'Different juices;',
+  'Different soups.',
+];
+
+const ulist = document.createElement('ul');
+for (let i = 0; i < meals.length; i += 1) {
+  const item = document.createElement('li');
+  item.textContent = meals[i];
+  ulist.appendChild(item);
+}
```
Comment (mikenath223) [inline src/menu.js]: - [ ] Replace this forLoop with a map function too. Do this for other functions too.

**Comment by ABDELLANI-Youcef** [comment]
Thanks very much for your advice. I made the requested changes.
Thanks again.

**Review by mosaaleb — APPROVED** [review]
#### Project Approved

Well done @ABDELLANI-Youcef :clap:. You have handled all the requirements from the previous review. 
You can merge :+1:

##### Muhammad Ebeid

**Pull request merged**